spec/models/thermometer_widget_spec.rb

```diff
@@ -1,41 +1,25 @@
 describe ThermometerWidget do
 
-  let(:content) { {
+  let(:params) { {
     goal: 10000,
     count: 500,
-    autoincrement: true
+    autoincrement: true,
+    page_display_order: 1
   } }
-  let(:string_content) { {
-    goal: "12345",
-    count: "45",
-    autoincrement: "0"
-  } }
```

Comment:
Yeah let's check. I don't think it's storing them as strings or this test never would have passed yesterday.